implementation/src/main/java/io/smallrye/config/lifecycle/ConfigSourceInterceptor.java

```diff
@@ -0,0 +1,7 @@
+package io.smallrye.config.lifecycle;
+
+import java.io.Serializable;
+
+public interface ConfigSourceInterceptor extends Serializable {
+    String getValue(ConfigSourceInterceptorContext context);
```

Comment:
This would change to:

```java
ConfigValue getValue(ConfigSourceInterceptorContext context, String name);
```

which allows an interceptor to aggregate multiple configuration values by calling `context.proceed()` multiple times.